collections/_news/2020-04-22-agm-2020-intro.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+published: true
+title: AGM 2020&colon; An Introduction
```

Comment:
You can put an actual colon here

Reply:
Are you sure? Didn't seem to work when I did it before - breaks the rendering of the post if there's a colon in the front matter.